src/driver/op.rs

```diff
@@ -190,10 +209,10 @@ mod test {
     impl Completable for Rc<()> {
         type Output = Completion;
 
-        fn complete(self, result: io::Result<u32>, flags: u32) -> Self::Output {
+        fn complete(self, cqe: CqeResult) -> Self::Output {
             Completion {
-                result,
-                flags,
+                result: cqe.result,
```

Comment:
Begs the question (for another day), looks like the Completion type can take this field too.